kuzzleio/kourou
# add support `--print-raw` to `sdk:query`

## What does this PR do?
Add flag `--print-raw` to command `sdk:query` to get only the result of query.
It's usefull to use the result of command in pipe with other CLI (like [jq](https://stedolan.github.io/jq/) for exemple)

### How should this be manually tested?

Just download this branch and try
`bin/run sdk:query index:list --print-raw`

The expected result should look like
```json
{
  "indexes": [
    "platform"
  ]
}
```
